components/authz-service/server/v2/project_test.go

```diff
@@ -489,8 +491,9 @@ func setupProjectsAndRules(t *testing.T) (api.ProjectsClient, *cache.Cache, *cac
 	l, err := logger.NewLogger("text", "error")
 	require.NoError(t, err, "init logger for storage")
 
-	mem_v2 := memstore_v2.New()
-	pg, testDb, _, _, _ := testhelpers.SetupTestDB(t)
+	mem_v2 := memstore_v2.NewWithProjectLimit(projectLimitForTesting)
+	// TODO: we should be able to optionally use PG. We're only using memstore
```

Comment:
@chef/auth-team  is that what we want?

Reply:
(we are testing the limit elsewhere, but I do think this suite *looks* like it's using PG when it's not . . .)

Reply:
I'd personally like us to ditch all the memstore tests altogether and just use PG where necessary but I might be in the minority there :)

Reply:
💯 These should either run both with memstore and pg or even just pg. Definitely worth making a tech debt card to discuss as a team. The only reason I can think of for using that we can easily seed the store with whatever we need for the test. But we can do that in pg too so it's not a great reason :/